Cloud changelog - Dec 8th (#6089)
mattermost/docs: source/install/cloud-changelog.md

Latest Mattermost Cloud releases:

- [Release 2022-12-08](#release-2022-12-08)
- [Release 2022-12-01](#release-2022-12-01)
- [Release 2022-11-24](#release-2022-11-24)
- [Release 2022-11-17](#release-2022-11-17)
- [Release 2022-11-10](#release-2022-11-10)
- [Release 2022-10-27](#release-2022-10-27)

## Release 2022-12-08

### Highlights

#### Message Priority and Acknowledgments (Professional plan)
 - Added message priority labels to the Threads view.
 - Added support for enterprise users to request acknowledgements on posts and to acknowledge posts.

#### Global Drafts

>  - Added a centralized page for draft messages.

Reviewer: @esethna - Since we have a Threads view, what do you think about us having a Drafts view too?

> Added ``acknowledgements`` field to the post's metadata.

Reviewer 2: missing a bullet

### Improvements

#### User Interface (UI)
 - Updated prepackaged version of plugins affected by React 17 upgrade.

#### Administration
 - The export file now contains the server version and a creation timestamp.

>  - The **System Console > Subscriptions** page now reflects if the plan is monthly or annual.

Reviewer: ```suggestion
 - The **System Console > Subscriptions** page now reflects whether the plan is monthly or annual.
```

>  - Added the ability to migrate from a monthly to an annual Cloud Professional plan.

Reviewer: in-product?

>  - On the purchase modal, admins are now able to buy a subscription count in excess of the current user count.

Reviewer: ```suggestion
 - On the purchase modal, admins are now able to buy a subscription for a user count that exceeds the current user count.
```

Latest Mattermost Cloud releases:

- [Release 2022-12-08](#release-2022-12-08)
- [Release 2022-12-01](#release-2022-12-01)
- [Release 2022-11-24](#release-2022-11-24)
- [Release 2022-11-17](#release-2022-11-17)
- [Release 2022-11-10](#release-2022-11-10)
- [Release 2022-10-27](#release-2022-10-27)

## Release 2022-12-08

### Highlights

#### Message Priority and Acknowledgments (Professional plan)
 - Added message priority labels to the Threads view.
 - Added support for enterprise users to request acknowledgements on posts and to acknowledge posts.

#### Global Drafts
 - Added a centralized Drafts view for draft messages.

### Improvements

#### User Interface (UI)
 - Updated prepackaged version of plugins affected by React 17 upgrade.

#### Administration
 - The export file now contains the server version and a creation timestamp.

>  - On the purchase modal, admins are now able to buy a subscription for a user count that exceeds the current user count.

Reviewer 3: This is also for Cloud Annual Subscription - we should include this as a highlighted new feature in the Cloud 15th release

Reviewer 4: Shouldn't we denote that the ability to buy a "subscription for a user count that exceeds the current user count" only for cloud annual and self-hosted annual? Cloud monthly does not have this capability

Author: I see, I'll remove this from this Cloud release for now and will add back for Dec 15th changelog.

 - Plugins with a webapp component may need to be updated to work with Mattermost and the updated ``React v17`` dependency. This is to avoid plugins crashing with an error about ``findDOMNode`` being called on an unmounted component. While our `starter template <https://github.com/mattermost/mattermost-plugin-starter-template>`_ depended on an external version of ``React``, it did not do the same for ``ReactDOM``. Plugins need to update their ``webpack.config.js`` directives to externalize ``ReactDOM``. For reference, see https://github.com/mattermost/mattermost-plugin-playbooks/pull/1489. Server-side only plugins are unaffected. This change can be done for existing plugins any time prior to upgrading to Mattermost and is backwards compatible with older versions of Mattermost.
 - Added ``acknowledgements`` field to the post's metadata.

### API Changes

>  - Added new endpoint ``api/v4/cloud/products/selfhosted``.

Reviewer 3: This is only applicable for self-hosted, we can remove from here

Latest Mattermost Cloud releases:

- [Release 2022-12-08](#release-2022-12-08)
- [Release 2022-12-01](#release-2022-12-01)
- [Release 2022-11-24](#release-2022-11-24)
- [Release 2022-11-17](#release-2022-11-17)
- [Release 2022-11-10](#release-2022-11-10)
- [Release 2022-10-27](#release-2022-10-27)

## Release 2022-12-08

### Highlights

#### Message Priority and Acknowledgments (Professional plan)
 - Added message priority labels to the Threads view.
 - Added support for enterprise users to request acknowledgements on posts and to acknowledge posts.

#### Global Drafts
 - Added a centralized Drafts view for draft messages.

### Improvements

#### User Interface (UI)
 - Updated prepackaged version of plugins affected by React 17 upgrade.

#### Administration
 - The export file now contains the server version and a creation timestamp.
 - The **System Console > Subscriptions** page now reflects whether the plan is monthly or annual.

>  - Added the ability to migrate from a monthly to an annual Cloud Professional plan in-product.

Author: I heard from Nick that these are not releasing yet, do I remove these from the changelog?

Reviewer 5: Confirmed, feature flagged, not going out this release. Can be removed.
